File: tasks/lectures/jnu/python-ai-basic/5th/5.py
arr = []
for i in range(5):
    arr += [int(input('성적을 입력하시오 : '))]
print('성적 평균은 {} 입니다.'.format(sum(arr)/5))
over_70 = 0
for i in arr:
    if i >= 70:
        over_70 += 1
print('70점 이상 성적을 받은 학생은 {} 명 입니다.'.format(over_70))

File: tasks/lectures/jnu/c-ce-basic-kim-2022-1/l-4.25/5.c
#include <stdio.h>

int main() {
  int arr[5] = { 10, 20, 30, 40, 50 };
  int* p = &arr[0];

  for (int i = 0; i < 5; i++, p++) {
    printf("%d\n", *p);
  }
}